src/builders/skopeo_builder.rs

```diff
@@ -460,6 +464,16 @@ impl SkopeoSyslinuxBuilder {
             .context("Failed to install rootfs from built container")
     }
 
+    /// Create `/workspace` directory in the VM, which will be used as output mountpoint.
+    fn create_workspace() -> Result<()> {
+        let ws_path = env::temp_dir().join("mnt").join("workspace");
+        if !ws_path.exists() {
+            Self::run_command(&["mkdir", "-p", ws_path.to_str().unwrap()], true)
```

Comment:
Can we just use the FS crate for this? I know we need to do it  this way when we want to use sudo, but this here is really overkill